tests/installation/await_install.pm

```diff
@@ -104,6 +104,10 @@ sub run {
             assert_screen \@tags, $timeout;
         }
 
+        if (match_has_tag('debugger_popup_warning')) {
+           die 'Ruby debugger popup warning detected.'
```

Comment:
It can be other kind of error which prevents us from finishing the installation, so what's about making it more generic, saying: `''YaST error detected`?